website/docs/r/integration_aws.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: "datadog"
+page_title: "Datadog: datadog_integration_aws"
+sidebar_current: "docs-datadog-resource-integration_aws"
+description: |-
+  Provides a Datadog - Amazon Web Services integration resource. This can be used to create and manage the integrations.
+---
+
+# datadog_integration_aws
+
+Provides a Datadog - Amazon Web Services integration resource. This can be used to create and manage Datadog - Amazon Web Services integration.
+
+Update operations are currently not supported with datadog API so any change forces a new resource.
```

Comment:
We should probably make this more evident and also explain the workaround of using `${account_id}_${role_name}` in the state in place of the real identifier.

Reply:
What do you mean by real identifier? There's no such an ID exposed via API. Should we manage IDs on our own in this case?

Reply:
Basically combination of an account ID and a role name should be unique for an integration, I can't think of a better way to implement ID.

Reply:
My bad, I was somehow convinced we returned an id in the response payload, please ignore my comment and let's stick with your plan

Reply:
Great, I guess there's nothing preventing a merge now. =)

Reply:
My only recommendation would be to use a `/` to separate account ID and Role name instead of `_`. That would be consistent with some of the multi-key import statements in the AWS provider. Example: https://www.terraform.io/docs/providers/aws/r/iam_role_policy_attachment.html#import

Well, on further investigation, it looks like they're anything but consistent...
https://github.com/terraform-providers/terraform-provider-aws/search?q=imported+separated+by+%2F&unscoped_q=imported+separated+by+%2F
If anything, I guess the `:` is the most common separator.

Reply:
Any non-numeric char could work here, changing to a colon now.

Reply:
Done.